### DISCUSSION TITLE
[question] ESM import bundle size

### DISCUSSION THREAD

**Discussion by sergchernata** [naver/billboard.js#3526] [General]
My import is as follows, bundled with webpack:
```
import bb, { spline } from "billboard.js"; // 3.10.3
```

With this import, my production bundle size increases by a little over 1MB. Does that sound right or am I missing something else?

**Comment by netil** [comment]
not the current v3 version, but the build size measured on v2, we got as follows results.
- https://github.com/naver/billboard.js/wiki/CHANGELOG-v2#example-of-size-reduction-by-types

The bundle size may differ according your webpack or usage configuration environment. 
And usually on webpack environment, non imported modules are tree-shaken to cut down from the bundling.
- https://webpack.js.org/guides/tree-shaking/
